base_iso3166/models/res_country.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# Copyright 2017  Creu Blanca <www.creublanca.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+try:
+    import pycountry
+except ImportError:
+    pass
```

Comment:
Why this here? If you added `pycountry` to the dependencies and to `requirements.txt` this should always work, right? Besides, will the module work without `pycountry` installed? If not, maybe it should give an error anyway...

Reply:
This comes from previous versions but I think that `try: .... except: ...` could be removed since there is no specific behavior defined for when the library is not found. Do you agree @pedrobaeza

Reply:
Yes, now it's safe to not protect imports, as Odoo handles this correctly.

Reply:
:+1: